pkg/kubectl/cmd/util/helpers.go

```diff
@@ -779,7 +779,7 @@ func DefaultSubCommandRun(out io.Writer) func(c *cobra.Command, args []string) {
 		c.SetOutput(out)
 		RequireNoArguments(c, args)
 		c.Help()
-		os.Exit(DefaultErrorExitCode)
+		CheckErr(fmt.Errorf("exit"))
```

Comment:
fyi, there's already a var defined for `fmt.Errorf("exit")` [on this file](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/util/helpers.go#L111) :)